Common/Product/SharedProject/Wpf/Controls.xaml

```diff
@@ -226,6 +226,10 @@
         <Setter Property="img:ImageThemingUtilities.ImageBackgroundColor" Value="{DynamicResource {x:Static wpf:Controls.InfoBackgroundColorKey}}" />
     </Style>
 
+    <Style x:Key="InfoText" TargetType="{x:Type TextBlock}" BasedOn="{StaticResource {x:Type TextBlock}}">
```

Comment:
Do we use this style anywhere right now?

Reply:
Yes, it's used in this PR in pipextension.xaml